.github/workflows/ci.yml

```diff
@@ -19,10 +18,10 @@ jobs:
           arch: amd64
 
       - name: Generate CMake project
-        run: cmake -D CMAKE_BUILD_TYPE=Release -S. -Bbuild -G Ninja
+        run: cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DOPTIMIZE=YES -S . -B build -G Ninja
```

Comment:
What does `-DOPTIMIZE=YES` do?
It's not used anywhere and cmake doesn't do anything with it internally.